test/E2ETests/E2ETests/E2ETests.csproj

```diff
@@ -19,6 +19,7 @@
       <PrivateAssets>all</PrivateAssets>
       <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
     </PackageReference>
+    <PackageReference Include="Azure.Data.Tables" Version="12.8.0" />
```

Comment:
You shouldn't have to do this; the worker extension will bring in this package

Reply:
I see [here](https://github.com/Azure/azure-functions-dotnet-worker/pull/1506/files/eda9939a70e1f0358eae19309ac72fcc25286867#diff-40f748aa75dcc9c5ab0e6e6354ca57e661c9d577a042ef0f2d383cac67ce1685R11) we're bringing in blob and it's not coming through the worker extension. Should table follow that pattern as well?